openhtf/util/units/__init__.py

```diff
@@ -0,0 +1,1170 @@
+# coding: iso-8859-1
```

Comment:
@grybmadsci, if I don't check in the generated Python, I'm going to need to either check in the .xls file, or download and unzip it from UNECE during the build step. I dislike the first because I don't like republishing someone else's document, and I dislike the second because of the potential volatility of UNECE URL's, .zip archive structure, and spreadsheet naming.

On the other hand, I think it's going to be tricky to make the generated code pass lint without exceptions. Would like to discuss more before moving code generation to build step.